.semaphore/semaphore.yml

```diff
@@ -4,33 +4,27 @@ agent:
   machine:
     type: e1-standard-2
     os_image: ubuntu1804
-
+global_job_config:
+  prologue:
+    commands:
+      - sem-version python 3.8
+      - sh ~/get-sam-cli.sh
+      - checkout
+      - cache restore requirements-$SEMAPHORE_GIT_BRANCH-$(checksum requirements.txt)
+      - pip install -r requirements.txt
```

Comment:
SHOULD: Just use `make init`. The semaphore files are good documentation for developers coming into a project.